handler.go

```diff
@@ -370,8 +464,8 @@ func (fm *Frontman) Run(inputFilePath string, outputFile *os.File, interrupt cha
 	for {
 		input, err := fm.FetchInput(inputFilePath)
 		if err != nil && err == ErrorMissingHubOrInput {
-			fmt.Println(err.Error())
-			os.Exit(1)
+			log.Warnln(err)
+			os.Exit(0)
```

Comment:
Why 0 and not 1?
Isn't 0 indicator for non-error?

Reply:
There should be no error, but a warning that Frontman has nothing to do. If no inputs are supplied, it can't do anything and quits. This allows successful installation and service check.